environment

```diff
@@ -1,4 +1,8 @@
 #!/usr/bin/env bash
 set -a
 PROJECT_ROOT=`pwd`
+AWS_DEFAULT_REGION='us-east-1'
+GOOGLE_APPLICATION_CREDENTIALS="${PROJECT_ROOT}/gcp-credentials-logs-travis.json"
```

Comment:
Setting `GOOGLE_APPLICATION_CREDENTIALS` might cause unexpected behavior. I'd avoid this if possible. Rather, expect the user to do so. You can put this in the `.gitlab-ci.yml`.

Reply:
👍